services/twitch/twitch.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const serverSecrets = require('../../lib/server-secrets')
+const { BaseJsonService, NotFound } = require('..')
+
+const schema = Joi.any()
```

Comment:
This should be built out to represent the API responses. If it's really necessary to make two API calls (see my question below) then we can define two schemas. 

Making a few guesses based on the existing code, but suspect you'd want something like these:

```js
const userSchema = Joi.object({
  data: Joi.array().required()
})

const streamsSchema = Joi.object({
  data: Joi.array().required()
})
```

Reply:
Actually, if the API responses are the same, then you could get by with a single schema:
```js
const schema = Joi.object({
  data: Joi.array().required()
})
```